hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,3 @@
+RELEASE_TYPE: patch
+
+Explicitly cast `finfo.smallest_normal` to builtin `float` in preparation for the `numpy=2.0` release (:issue:`3950`)
```

Comment:
```suggestion
Explicitly cast :obj:`numpy.finfo.smallest_normal` to builtin `float` in
preparation for the :pypi:`numpy==2.0 <numpy>` release (:issue:`3950`)
```

Annoyingly, rst and markdown have different semantics for backticks - suggestion above will turn them into semantic links; double-backticks will render as inline monospace/code font.